sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcIO.java

```diff
@@ -39,6 +39,7 @@
 import java.util.Collection;
```

Comment:
Should we update the documentation of the IO to specifically say the lineage metrics are only collected in these cases to set customer expectations correctly.

Reply:
added documentations for JdbcUtil.JdbcUrl.of. Was hesitate on adding the information to top level JdbcIO javadoc, as it is usually designated for instruction for read and write transform.